src/courtroom.cpp

```diff
@@ -4522,6 +4522,7 @@ void Courtroom::on_sfx_context_menu_requested(const QPoint &pos)
 
   menu->setAttribute(Qt::WA_DeleteOnClose);
   menu->addSeparator();
+  menu->addAction(QString("Play"), this, SLOT(on_sfx_play_clicked()));
```

Comment:
```suggestion
  menu->addAction(QString("Play"), this, &Courtroom::on_sfx_play_clicked);
```
We should use the Qt5 style where possible to take advantage of the compiler validation.

Reply:
I'm just following the previous code style that was used.

Reply:
Yeah, that one is on me. Thought I had caught them all in my prior run trough the codebase.